server/index.js

```diff
@@ -1,12 +1,14 @@
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
-const router = require('./routes/index.js');
-const db = require('../db');
+const detail = require('./routes/detail.js');
+const userRequest = require('./routes/userRequest.js');
+require('dotenv').config();
```

Comment:
This can be moved to the top of the file. I think the linter will complain about order of your requires.